Documentation/api-reference.md

```diff
@@ -73,6 +99,18 @@ public static void registerExtension()
 Consent.registerExtension();
```

Comment:
can you also update the registerExtensions link above to point to https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/api-reference.md